src/index.html

```diff
@@ -1,14 +1,44 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div id="qa-block" class="external">
+      <section>
+        <p class="inner">
+          In programming languages, a closure is a technique for implementing lexically scoped name binding in a
+          language with first-class functions. Operationally, a closure is a record storing a function together with an
+          environment.
+        </p>
+      </section>
+    </div>
+
+    <div class="distance"></div>
+
+    <div class="external">
+      <section>
+        <p class="inner">
+          У програмуванні замиканням називають підпрограму, що виконується в середовищі, яке містить одну або більше
+          зв’язаних змінних. Підпрограма має доступ до цих змінних під час виконання.
+        </p>
+      </section>
+    </div>
+
+    <div class="distance"></div>
+
+    <div class="external">
+      <section>
```

Comment:
let's add class to section too and style this element by class

Reply:
I add class, check pls)